src/build_loop.rs

```diff
@@ -95,9 +95,12 @@ impl<'a> BuildLoop<'a> {
                         output_paths = Some(result.output_paths.clone());
                         send(Event::Completed(result));
                     }
-                    Err(e) if e.is_actionable() => send(Event::Failure(e)),
                     Err(e) => {
-                        panic!("Unrecoverable error:\n{}", e);
+                        if e.is_actionable() {
```

Comment:
These changes were necessary because of the downgrade from Rust 1.40 in `nixos-unstable` to Rust 1.37 in `nixos-19.09`.